src/component/Missions.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styles from '../styles/Missions.module.css';
+import { getMission, joinMission } from '../features/missions/missionSlice';
+
+function Missions() {
+  const { missions } = useSelector((store) => store.missions);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMission());
+  }, [dispatch]);
```

Comment:
- [ ] Kindly move this code to `App.js` as now when I press the `Join Mission` button and go to the profile page I see it registered correctly, but, when I revisit the missions page and go to the profile page again I find no mission in profile page as the mission fetched every time the mission page requested.

Reply:
Thanks for this review, I understand what you have requested and would make the changes accordingly.